index.js

```diff
@@ -3,35 +3,59 @@
 
 import element from 'magic-virtual-element';
 
+let intervals = {};
+let hoverStates = {};
```

Comment:
@iefserge I'd say no since we'd then need to add a dependency (`es6-map`) but an object works just fine here..